pages/my-items.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, useEffect, useState } from 'react';
```

Comment:
Can you rename the detail page from `my/[templateId]` to `my-items/[templateId]` to match this `my-items` route?